pom.xml

```diff
@@ -1341,6 +1341,14 @@
                     <bannedImport>org.glassfish.jersey.**</bannedImport>
                   </bannedImports>
                 </restrictImports>
+                <restrictImports implementation="de.skuzzle.enforcer.restrictimports.rule.RestrictImports">
+                  <includeTestCode>true</includeTestCode>
+                  <commentLineBufferSize>512</commentLineBufferSize>
+                  <reason>Use RestrictedApi annotation in error prone instead</reason>
```

Comment:
HBASE-24640 replaced VisibleForTesting with IA.Private annotation. The `reason` in this `restrictImports` should recommend the same.

Reply:
I do not think this is the case. Even in IA.Public classes, not all the IA.Private annotations are for testing, and here we are telling a more general rule, not only for IA.Public classes. Saying that use IA.Private instead of VisibleForTesting will make developers confusing.
Let me see if I could make things more clear.

Reply:
I changed the wording. PTAL. Thanks.